spec/acceptance/suites/ds389/nodesets/default.yml

```diff
@@ -0,0 +1,47 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
+    hypervisor = ENV['BEAKER_HYPERVISOR']
+  else
+    hypervisor = 'vagrant'
+  end
+-%>
+HOSTS:
+  ldapserver:
+    roles:
+      - ldap
+      - default
+    platform: el-8-x86_64
+    box: centos/8
+    hypervisor: <%= hypervisor %>
+  centos8:
+    roles:
+      - client
+      - sssdv2
+    platform: el-8-x86_64
+    box: centos/8
+    hypervisor: <%= hypervisor %>
+  centos7:
+    roles:
+      - client
+      - sssdv1
+    platform: el-7-x86_64
+    box: centos/7
+    hypervisor: <%= hypervisor %>
+CONFIG:
+  log_level: verbose
+  type: aio
+  vagrant_memsize: 256
+<% if ENV['BEAKER_PUPPET_ENVIRONMENT'] -%>
+  puppet_environment: <%= ENV['BEAKER_PUPPET_ENVIRONMENT'] %>
+<% end -%>
+  ssh:
+    keepalive: true
+    keepalive_interval: 10
+    host_key:
+      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:host_key].join("\n#{' '*6}- ") %>
```

Comment:
Question:  Do we need the host_key, key, encryption, and hmac settings here anymore?  I thought this was put in place because of problems testing with CentOS 6.